gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -177,6 +177,18 @@ case class TransformPreOverrides() extends Rule[SparkPlan] {
             plan.initialInputBufferOffset,
             plan.resultExpressions,
             child)
+      case plan: ObjectHashAggregateExec =>
```

Comment:
should validate in  `AddTransformHintRule` before transform

Reply:
Thanks, fixed.

Reply:
I know, this is also why CH test failed, because collect test is ObjectHashAggregateExec, which fallbacked before, so I still need to add collect_list to black expression list